docs/Developers/Rankings.rst

```diff
@@ -0,0 +1,103 @@
+Rankings
+========
+
+The coala community is split into different ranks which have different rights
+and obligations. This page will explain which the ranks are and what you need
+to do to get promoted to a higher rank.
+
+Currently the ranks can be seen `here <https://github.com/orgs/coala/teams>`__
+and are as it follows:
+
+ - newcomers
+ - developers
+ - maintainers
+ - admins
+
+The Newcomer Rank
+-----------------
+
+Everyone starts here. This is the rank where you can get a promotion easiest
+and which has the lowest obligations, and, obviously, the lowest rights.
+
+Rights
```

Comment:
Issues can be assigned to them. That was the whole point of org membership. So that is a right I feel.